docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/TTL.md

```diff
@@ -1,79 +1,111 @@
 # TTL (time-to-live)
 
-With **TTL**, **Nebula Graph** provides the ability to filter the expired data automatically when traversing vertices and edges. The system will automatically delete the expired data during the compaction phase, without a delete operation that is explicitly issued by the console.
+With **TTL**, **Nebula Graph** provides the ability to delete the expired vertices or edges automatically. The system will automatically delete the expired data during the compaction phase. Before compaction, query will filter the expired data.
```

Comment:
Before compaction, the expired data will be ignored by queries

Reply:
Before compaction, query will filter the expired data. And the TTL data is not available.